src/main/java/org/myrobotlab/service/Hd44780.java

```diff
@@ -220,6 +231,7 @@ private void lcdWrite(byte cmd) {
   private void lcdWrite(byte cmd, byte mode) {

```

Comment:
maybe this should be synchronized ? is it possible that multiple threads could call this and screw up the data...